app/src/common/types.ts

```diff
@@ -7,12 +7,19 @@ export enum EventType {
 	DEPLOYMENT = 'deployment'
 }
 
+export enum ConnectedState {
+	CONNECTED = 'CONNECTED',
+	DUPLICATE = 'DUPLICATE',
+	PENDING = 'PENDING'
+}
+
 export interface JenkinsServer {
 	uuid: string,
 	name: string,
 	secret?: string,
 	pipelines: JenkinsPipeline[],
 	pluginConfig?: JenkinsPluginConfig;
```

Comment:
existing issue on this line, `pluginConfig?: JenkinsPluginConfig;` can we replace the `; `with a` , `

Reply:
Oh yeah, that was stoopid of me wasn't it?

Reply:
Surprised the linter didn't complain coz it shits the bed usually over the smallest things